livekit-agents/livekit/agents/voice_assistant/agent_output.py

```diff
@@ -34,13 +34,16 @@ def __init__(
         self._buf_ch = utils.aio.Chan[rtc.AudioFrame]()
         self._play_handle: PlayoutHandle | None = None
         self._interrupt_fut = asyncio.Future[None]()
-        self._collected_text = ""  # collected text from the async stream
         self._speech_id = speech_id
 
     @property
     def speech_id(self) -> str:
         return self._speech_id
 
+    @property
+    def tr_fwd(self) -> agent_transcription.TTSSegmentsForwarder:
```

Comment:
make this more understandable in the public API at least

```suggestion
    def transcription_forwarder(self) -> agent_transcription.TTSSegmentsForwarder:
```

Reply:
good point, +1 making the public API clear vs abbreviated

Reply:
this isn't public, this is still internal

Reply:
maybe `tts_forwarder` ?